Gemfile

```diff
@@ -91,12 +91,12 @@ group :development, :test do
   gem 'aws-sdk-cloudwatchlogs', require: false
   gem 'brakeman', require: false
   gem 'bullet', '~> 7.0'
-  gem 'capybara-webmock', git: 'https://github.com/hashrocket/capybara-webmock.git', ref: '63d790a0'
-  gem 'data_uri', require: false
+  gem 'capybara-webmock', git: 'https://github.com/mitchellhenke/capybara-webmock.git',
+                          ref: 'c23b02f'
```

Comment:
for posterity, [changes captured in this PR](https://github.com/hashrocket/capybara-webmock/pull/50)

Reply:
the PR has been merged, so I switched back to their repo in our Gemfile